core/metacling/src/TCling.cxx

```diff
@@ -7413,6 +7427,22 @@ Bool_t TCling::LoadText(const char* text) const
    return (fInterpreter->declare(text) == cling::Interpreter::kSuccess);
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Make RAII for redirecting diagnostic messages to an ostream
```

Comment:
Perhaps consider extending the documentation, incl. argument descriptions and/or usage example.